html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label> <br />
+      <input type="text" name="email" size="15" maxlength="50"> <br />
+
+    <label for="coding">Why is coding cool?</label> <br />
+      <select name="coding">
+        <option value="select">Select One</option>
+        <option value ="optionone">Because its intersting</option>
+        <option value="optiontwo">Because its fun</option>
+        <option value="optionthree">Because its powerful</option>
+      </select> <br />
+
+    <label for="language">Favorite Language (choose one)<br />
+      <input type="radio" name="language" value="html" />
+        HTML
```

Comment:
This text should be the `label` for the input it is paired with, since it is the text the user would click to put focus on the input it is labelling.